src/templates/tr/base.html

```diff
@@ -0,0 +1,184 @@
+{% extends "base/base.html" %}
+
+{% block description %}Web Almanak, web topluluğunun uzmanlığını HTTP Archieve verileri ve eğilimleri ile birleştiren yıllık bir web raporudur.{% endblock %}
+
+{% block twitter_image_alt %}{{ year }}Web Almanak{% endblock %}
+
+{% block skip_navigation %}Navigasyonu Atla{% endblock %}
+
+{% block organization %}Web Almanak HTTP Archive&#8217;dan{% endblock %}
+{% block web_almanac_logo %}
+<span class="wa">Web Almanak</span>
+<span class="line-group">
+  <span class="ha">HTTP Archive</span>
+  <span class="pre">by</span>
```

Comment:
Should "by" be translated?

This creates the logo you can see in the top right of each page: https://almanac.httparchive.org/en/2020/

Japanese has completed rearranged this as the words should be in a completely different order for that language: https://almanac.httparchive.org/ja/2020/. So let me know if we need something funky like that for Turkish.

Reply:
It has also different word order for Turkish.

Reply:
This is what it looks like when site is launched:

![Turkish Logo](https://user-images.githubusercontent.com/10931297/113271547-91182c00-92d2-11eb-896c-e1fc47b4caf8.png)

Happy with that? Or does it need any further changes?

Reply:
This is the right word order. No need any changes.